src/components/Form.tsx

```diff
@@ -18,7 +18,12 @@ import {
 } from "../utils";
 import validateFormData, { toErrorList } from "../validate";
 
-export default class Form extends Component {
+interface IFormProps {
```

Comment:
Has there been any discussion to export typings with the build or will you maintain typings within [@types/react-jsonschema-form](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/a819fdd3bc950160780f1e6e69c55a2bac8940f0/types/react-jsonschema-form)?

To add onto that, has there been any discussion regarding naming conventions for types?

Reply:
Not yet. It would be great to start one!

What would be the advantages/disadvantages of exporting typings with the build vs maintaining typings within @types/react-jsonschema-form? Do other libraries based in typescript typically do the former?

Reply:
> What would be the advantages/disadvantages of exporting typings with the build vs maintaining typings within @types/react-jsonschema-form?

A big drawback with using DefinitelyTyped is that it's community maintained and can sometimes get out of sync with the actual library. That repository is mainly for JS packages that don't have typings for TS projects. And sometimes DefinitelyTyped can take a while to merge PR's so you'd have to wait for a release before you could use new typings. 

With that in mind, it's usually better to export typings with the package so your types are always 1:1 and you don't have to maintain two separate packages.

> Do other libraries based in typescript typically do the former?

Usually yeah. Check out [nestjs](https://github.com/nestjs/nest) or even vscode. Go explore some more TS packages here in [GH](https://github.com/topics/typescript)